externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/ExecutePython.scala

```diff
@@ -167,6 +172,11 @@ class ExecutePython(
       }
     }
   }
+
+  override protected def setState(newState: OperationState): Unit = {
```

Comment:
Why not override method `setState` on the superclass `SparkOperation`, then all the operations can be seen in the tab. cc @cxzl25

Reply:
It sounds pretty good.

This PR should also fix https://github.com/apache/kyuubi/issues/2886